packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js

```diff
@@ -253,4 +256,10 @@ export default class OvhPaymentMethodIntegrationComponentAdyenCtrl {
       callbackParamObject,
     );
   }
+
+  getEnvironment() {
+    return this.user.ovhSubsidiary === IN_SUBSIDIARY
```

Comment:
Will another subs will added later ? Maybe you should consider using feature flipping instead ?
What do you think ?

Reply:
May be not as the environment is only for IN, but I will consider your suggestion and add feature flipping.

Reply:
Its done